assignmentPR/conversionF1.c

```diff
@@ -74,6 +48,28 @@ int characteristic(char numString[], int* c)
 /*___________________________________________________________________________*/
 
 
+/*___________________________________________________________________________*/
+//Converts character into an integer
+int charToInt(char* charArray)
+{
```

Comment:
I recommend you look into upgrading your indentation style. The "One True Brace" method is popular and easy to learn.

Reply:
@raptorace27 @jbell-97 @tanman987 Highlighting this comment